src/widget/emoticonswidget.h

```diff
@@ -31,10 +31,12 @@ class EmoticonsWidget : public QMenu
 private slots:
     void onSmileyClicked();
     void onPageButtonClicked();
+    void PageButtonsUpdate();
 
 protected:
     virtual void mouseReleaseEvent(QMouseEvent *ev);
     virtual void mousePressEvent(QMouseEvent *ev);
+    virtual void wheelEvent(QWheelEvent * event) Q_DECL_OVERRIDE;
```

Comment:
Please just use `override` instead of  `Q_DECL_OVERRIDE`, no need to use macros since we require a C++11 compiler anyways.